chimera_app/platforms/gog.py

```diff
@@ -51,13 +48,11 @@ def get_shortcut(self, content):
 
     def get_installed_content(self) -> list:
         games = self.__get_all_content()
-        return [game for game in games if game.installed == True]
-
+        return [game for game in games if game.installed]
 
     def get_available_content(self) -> list:
         games = self.__get_all_content()
-        return [game for game in games if game.installed == False]
-
+        return [game for game in games if game.installed]
```

Comment:
this will never show games that are not installed

Reply:
Oops. There should be a `not` in there